components/camera/camera.go

```diff
@@ -136,6 +159,22 @@ func ReadImage(ctx context.Context, src gostream.VideoSource) (image.Image, func
 	return gostream.ReadImage(ctx, src)
 }
 
+// ImageFromVideoSource retrieves image bytes from a camera source and decodes it into an image.Image type.
+func ImageFromVideoSource(ctx context.Context, mimeType string, extra map[string]interface{}, cam VideoSource) (image.Image, error) {
```

Comment:
Why is `cam` a `VideoSource` and not a `Camera`?

Reply:
Done. It was not a big lift to fix tests in response to changing VideoSource to Camera